### PR TITLE
Update TESTS.md (dart test should also be a command people should know about)

### PR DESCRIPTION
**Pull request by MSoup**
I noticed that many people have had problems using pub. I added a little excerpt about how `dart test` also works, and is possibly more beginner friendly.

### CODE REVIEW

File: docs/TESTS.md [hunk 1]
```diff
@@ -29,6 +29,19 @@ It is advised to enable tests one by one as you implement the functionality bein
...
+NOTE: Some users may run into the error:
+
+```
+command not found: pub
+```
+
+If your terminal does not know where pub is, you may need to add it to your PATH. Alternatively, try the following command from the root directory of the exercise:
+
+```
+dart test
```
Comment (Stargator) [inline docs/TESTS.md:41]: I suggest updating all the existing commands to use the newer structure. Like replace `pub run test` with `dart test`.

**Review by Stargator — CHANGES_REQUESTED** [review]

**Review by MSoup — COMMENTED** [review]
I think this should do it. There is a HELP.md that comes with each exercise that isn't available in here. Do you happen to know where I can access it?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Install dependencies
-      run: dart pub get
+      run: dart dart pub get
```
Comment (Stargator) [inline .github/workflows/test.yml]: No need to duplicate the `dart` command. That won't work in GitHub Actions.

I think you just need a copy/paste, so just need to correct this.

**Review by Stargator — CHANGES_REQUESTED** [review]
I correction and I think this would be good.

File: .github/workflows/test.yml (hunk 2 above)
Comment (MSoup) [reply .github/workflows/test.yml]: My apologies, I caught that mistake and did a second commit that reflects that change.

**Review by Stargator — CHANGES_REQUESTED** [review]

File: exercises/shared/.docs/tests.md [hunk 3]
```diff
@@ -3,5 +3,5 @@
 To run the tests:
 
 ```sh
-$ pub run test
+$ dart run test
```
[the remark this answers is not in the document]
Comment (MSoup) [reply exercises/shared/.docs/tests.md]: Apologies for the delay. Does my latest commit seem okay?

**Review by Stargator — APPROVED** [review]

**Comment by Stargator** [comment]
@MSoup Sorry for the delay, I am running the Actions workflow against this and if good, I'll merge it

**Pull request merged**